app/src/main/java/es/situm/gettingstarted/drawbuilding/DrawBuildingActivity.java

```diff
@@ -49,10 +65,10 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     @Override
-    public void onMapReady(GoogleMap googleMap) {
+    public void onMapReady(@NonNull GoogleMap googleMap) {
         map = googleMap;
 
-        fetchFirstFloorImage(building, new Callback() {
+        fetchFloorsImages(building, new Callback() {
             @Override
             public void onSuccess(Bitmap floorImage) {
                 drawBuilding(floorImage);
```

Comment:
Diría que `drawBuilding` xa non é "draw building", non? Máis ben sería un "draw floor" ou algo así?